bundles/AdminBundle/Controller/Admin/LogController.php

```diff
@@ -229,10 +229,13 @@ public function showFileObjectAction(Request $request)
         $storage = Storage::get('application_log');
 
         if ($storage->fileExists($filePath)) {
-            $fileData = $storage->readStream($filePath);
+            $fileHandle = $storage->readStream($filePath);
             $response = new StreamedResponse(
-                static function () use ($fileData) {
-                    echo stream_get_contents($fileData);
+                static function () use ($fileHandle) {
+                    while (!feof($fileHandle)) {
+                        echo fread($fileHandle, 8192);
+                    }
+                    fclose($fileHandle);
```

Comment:
@BlackbitDevs thank you for the pull request. One small suggestion since now are contents of responses from lines 234 - 242 almost identical with 261 - 271. Could you extract these to a separate helper service, where you pass the file contents. 
<img width="476" alt="loggingFileResponse" src="https://user-images.githubusercontent.com/30526586/211826312-c0fc4af4-51bb-4aea-b8fa-3245be9c7191.png">

Reply:
I have put the code into a private method.

Reply:
Perfect thank you